src/main/java/duke/DeadlineCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class DeadlineCommand extends Command{
+
+    @Override
+    void execute(String fullCommand, ArrayList<Task> listOfTasks, Ui ui, Storage storage) throws IOException, DukeDeadlineEmptyException {
+        if(fullCommand.length() == 8){
```

Comment:
Do you think there should have been whitespaces to separate the `if`, the condition, and the curly brace?